source/deployment-options/splunk/splunk-app.rst

```diff
@@ -139,9 +147,11 @@ Install the Wazuh app for Splunk in a minimal distributed architecture
 
 #. The app will redirect you to the Settings tab. If you are not redirected to the Settings tab, select the settings icon, this will take you to the “Settings” page.
 
-   .. thumbnail:: /images/splunk-app/16.png
+   .. thumbnail:: /images/splunk-app/settings-tab.png
```

Comment:
This is more the Settings "icon" and not the Settings tab

Reply:
You are right, and I'll change it!